hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -226,6 +226,24 @@
         - 'kubernetes-e2e-{suffix}'
 
 - project:
+    name: kubernetes-e2e-gce-1-2
+    trigger-job: 'kubernetes-build-1.2'
```

Comment:
should we also set `branch`, even though it's not used by e2e jobs as best I can tell?